wire-library/wire-compiler/src/test/java/com/squareup/wire/WireCompilerErrorTest.kt

```diff
@@ -140,6 +140,7 @@ class WireCompilerErrorTest {
           |multiple enums share constant QUIX:
           |  1. com.squareup.protos.test.Bar.QUIX (/source/test.proto:4:3)
           |  2. com.squareup.protos.test.Bar2.QUIX (/source/test.proto:10:3)
+          |  for file /source/test.proto
```

Comment:
That's unfortunate.
I cannot come up with a fix for this specific case, and I also think it's not a big of a deal with we add this line for this error?

Reply:
Seems fine